### PR TITLE
AP_HAL: Set default params for Bebop

### PR DESCRIPTION
**Pull request by jberaud**

### CODE REVIEW

File: libraries/AP_HAL/AP_HAL_Boards.h [hunk 1]
```diff
@@ -229,6 +229,7 @@
 #define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
 #define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
 #define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_PARAM_DEFAULTS_PATH "/etc/arducopter/bebop.parm"
```
Comment (lucasdemarchi) [inline libraries/AP_HAL/AP_HAL_Boards.h]: I'd like to start thinking about standardizing this location. I think we need a dir in /etc that we can put everything. With the new build system we would even be able to "make install" to install the right files like other programs do.

I propose we use `/etc/apm` or `/etc/ardupilot`. Imagining more vehicles can be installed at the same time, but it's intended to run on the same board, I think `/etc/ardupilot/copter.parm` would be sufficient. What do you think?

**Comment by magicrub** [comment]
I see that PX4 uses "/etc/defaults.parm" and bebop and SITL use "etc/defaults.parm" without the root slash. Is that intentional?

File: libraries/AP_HAL/AP_HAL_Boards.h (hunk 1 above)
Comment (jberaud) [inline libraries/AP_HAL/AP_HAL_Boards.h]: I agree that we could choose a standard path for this file. I am already using our build system to copy the file at startup with a config to choose between bebop 1 and 2 and that you can see here:
https://github.com/Parrot-Developers/ardupilot/blob/master-alchemy-build/atom.mk#L42
I don't have any objection about the exact path, /etc/apm seems fine. We could accept this PR and then modify this in order to be the same on all linux boards.

**Comment by jberaud** [comment]
@magicrub : bebop uses /etc/arducopter/bebop.parm

**Comment by lucasdemarchi** [comment]
@magicrub unless they do a chdir to /, no it doesn't make sense.

File: libraries/AP_HAL/AP_HAL_Boards.h (hunk 1 above)
Comment (lucasdemarchi) [inline libraries/AP_HAL/AP_HAL_Boards.h]: ok

**Comment by lucasdemarchi** [comment]
Applied. Thanks.

**Pull request closed without merging**